.github/workflows/connector_ops_ci.yml

```diff
@@ -21,3 +22,35 @@ jobs:
         run: pip install --quiet -e ./tools/ci_connector_ops
       - name: Check test strictness level
         run: check-test-strictness-level
+  requires-connector-team-review:
+    name: "Check if a review is required from Connector teams"
+    runs-on: ubuntu-latest
+
+    # GitHub should provide a "pull_request_review_target", but they don't and
+    # the action will fail if run on a forked PR.
+    if: github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+      - name: Install ci-connector-ops package
+        run: pip install --quiet -e ./tools/ci_connector_ops
+      - name: Check if the PR requires a connector team review
+        id: check-requires-connector-team-review
+        run: check-requires-connector-team-review >> $GITHUB_OUTPUT
+      - name: Check if the required teams reviewed
+        if: steps.check-requires-connector-team-review.outputs.REQUIRES_CONNECTOR_TEAM_REVIEW == 'True'
+        uses: Automattic/action-required-review@v3
+        with:
+          status: Required review
+          fail: true
+          token: ${{ secrets.OCTAVIA_4_ROOT_ACCESS }}
+          requirements: |
+            - paths: unmatched
+              teams:
+                - connector-operations
```

Comment:
[non-blocking] Do you think that we are /always/ the proper people to take the review?  On one hand, we know the most about the spec and possibly the CDK (+connector-extensibility?), but perhaps the question is connector specific and we will need GL...

I guess we can re-assign the review as needed.

Reply:
A cool evolution would be to make the python script return the name of the teams that should review. To keep things simple at the moment, I'll make both `connector-operations` and `connector-extensibility` required reviewers (in an `any-of` manner).